papermill/execute.py

```diff
@@ -229,14 +229,31 @@ def _find_parameters_index(nb):
         raise PapermillException("Multiple parameters tags found")
     return parameters_indices[0]
 
-def _form_escaped_value(str_val):
+def _form_escaped_str(str_val):
     if isinstance(str_val, string_types):
         str_val = str_val.encode('unicode_escape')
         if sys.version_info >= (3, 0):
             str_val = str_val.decode('utf-8')
         str_val = str_val.replace('"', r'\"')
     return '"{}"'.format(str_val)
 
+def _form_escaped_r_value(val):
```

Comment:
Something about `_form_escaped_...` feels a bit awkward as a name (probably because form can be a noun or a verb, same with escape). Can we simplify to  `_translate_type`, `_translate_str_r`, `_translate_value_r`,  `_translate_dict_r`...? This way if more languages are added the pattern remains consistent `_translate_{type}_{language}`.

Reply:
Sure, might be a little verbose that way but clear on extensibility and expectations.